app/src/main/AndroidManifest.xml

```diff
@@ -74,7 +74,7 @@
                 android:name=".ui.WireActivity"
                 android:exported="true"
                 android:hardwareAccelerated="true"
-                android:launchMode="singleTop"
+                android:launchMode="singleTask"
```

Comment:
this change may interfere with deep links, please check out if they still work as expected with this change

Reply:
Tested it briefly with the server config deeplink and the one to join a given conversation and all seems to work correctly.